src/features/island/buildings/components/building/market/Seeds.tsx

```diff
@@ -138,6 +138,7 @@ export const Seeds: React.FC<Props> = ({ onClose }) => {
 
   const labelState = () => {
     const max = INITIAL_STOCK[selectedName];
+    const inventoryFull = max ? inventory[selectedName]?.gt(max) ?? true : true;
```

Comment:
```suggestion
    const max = INITIAL_STOCK[selectedName];
    const inventoryCount = inventory[selectedName] ?? new Decimal(0);
    const inventoryFull = max ? inventoryCount.gt(max) : true;
```

The seed shop was showing red labels when I had no seeds in my inventory

<img width="341" alt="Screenshot" src="https://user-images.githubusercontent.com/41215134/205522273-1227d85c-bcf4-4d3f-801f-fed8004fca31.png">